### PR TITLE
feat(Consents): Remove old actions required form.

### PR DESCRIPTION
**Pull request by lauryndbrown**
Part of the work that fixes https://github.com/carpentries/amy/issues/1881.

Removes the unused actions required form in favor of now only using the new consents based form.

### CODE REVIEW

File: amy/api/tests/test_api_structure.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from django.urls import reverse
 from rest_framework.test import APITestCase
 
+from amy.workshops.tests.base import consent_to_all_required_consents
```
Comment (pbanaszkiewicz) [inline amy/api/tests/test_api_structure.py:4]: Please fix to `workshops.tests.base` (without `amy.`).

File: amy/workshops/urls.py [hunk 2]
```diff
@@ -4,86 +4,171 @@
 
 urlpatterns = [
     # utility views
-    path('log/', views.changes_log, name='changes_log'),
-    path('version/<int:version_id>/', views.object_changes, name='object_changes'),
-    path('workshop_staff/', views.workshop_staff, name='workshop_staff'),
-    path('workshop_staff/csv/', views.workshop_staff_csv, name='workshop_staff_csv'),
-
+    path("log/", views.changes_log, name="changes_log"),
```
Comment (pbanaszkiewicz) [inline amy/workshops/urls.py:7]: I guess the GH linter workflow didn't allow you to leave the file styling untouched? Unfortunately I didn't fix it in time to ignore `urls.py`.

But on the other hand, it's just easier to keep no exceptions from Black, and the formatted file is still quite readable.

**Review by pbanaszkiewicz — APPROVED** [review]
Looks good to me (except the imports).

File: amy/workshops/urls.py (hunk 2 above)
Comment (lauryndbrown) [reply amy/workshops/urls.py:7]: @pbanaszkiewicz that's right. I couldn't pass the linter it without making an exception for this file.

File: amy/api/tests/test_api_structure.py (hunk 1 above)
Comment (lauryndbrown) [reply amy/api/tests/test_api_structure.py:4]: Oh hmm.. thought I'd fixed that in my editor. Good catch.

**Pull request merged**